target/linux/ar71xx/image/mikrotik.mk

```diff
@@ -26,12 +26,12 @@ TARGET_DEVICES += nand-64m nand-large
 
 define Device/rb-nor-flash-16M
   DEVICE_TITLE := MikroTik RouterBoard with 16 MB SPI NOR flash
-  DEVICE_PACKAGES := rbcfg rssileds
+  DEVICE_PACKAGES := rbcfg rssileds ath10k-firmware-qca9887 kmod-ath10k
```

Comment:
So, again back to roots, I have here same question as in https://github.com/lede-project/source/pull/925#discussion_r104659643: do all below boards need both the ath10k firmware and the driver? Well, we all know the answer.

Argument for keeping `rssileds` package for all these boards was, more or less, like _it's just a few KB and all these boards have 16 MB_ - fine, I agree with this after considering all pros and cons. But this time it's not _just a few KB_, I simply don't like this approach.

@blogic, @mkresin, @nbd168, what do you think about this?

@rmounce, please rebase on current master and don't include in your PR commits which are not yet in the repository.